nengo_viz/components/sim_control.py

```diff
@@ -86,9 +86,11 @@ def update_client(self, client):
             client.write('reload')
             self.reload = False
         if self.send_config_options == True:
-            from ..disposable_js import configmodal
-            javascript = configmodal()
-            client.write('config' + javascript)
+            options = ["zoom"] #This is ignored by the client for now
+                               #But will be useful when saving/loading options
+            js_options = json.dumps(options)
```

Comment:
I would prefer to exclude such code in the merge (we can keep it in a separate branch).